docs/getting-started/README.md

```diff
@@ -4,48 +4,25 @@
 
 Frontity is a **React-based framework** that enables you to easily build a frontend for a headless \(or decoupled\) WordPress site. Your WordPress site serves its data via the REST-API. Frontity framework is open source and free to use.
 
-## Table of Contents
-
-<!-- toc -->
-
-- [Requirements](#requirements)
-  * [WordPress](#wordpress)
-    + [Have a recent WordPress version](#have-a-recent-wordpress-version)
-    + [Set the right REST API URL](#set-the-right-rest-api-url)
-    + [Have pretty permalinks activated](#have-pretty-permalinks-activated)
-  * [Node.js](#node-js)
-    + [Have Node.js and npm installed](#have-node-js-and-npm-installed)
-- [Get up and running](#get-up-and-running)
-
-<!-- tocstop -->
-
 ## Requirements
 
 To get started with Frontity you will need:
 
-* **A WordPress installation** - This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com
-
-* **Node.js** - If you don't already have it you can get *Node.js* from [the official site](https://nodejs.org/). This will also install `npm` and `npx` along with Node.js
-
-  You will use these to run Frontity commands during the set-up and development of your project
-
-
-{% hint style="info" %}
-For those coming from WordPress it might be worth noting that *Frontity* runs on **Node.js**, so it needs to be deployed in a different server than your WordPress. If you want to learn more about this, visit our [GitHub repo](https://github.com/frontity/frontity#why-a-different-nodejs-server) or see the [Architecture](../architecture.md) section arch these docs.
+### A WordPress installation
 
-{% endhint %}
+Frontity provides a [default WordPress site](https://test.frontity.org/) as source of data in every new project so for this Quick Start Guide we can work with this one
 
-### WordPress
+You can also [set your own WordPress installation](use-your-wordpress-installation.md). This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com. 
```

Comment:
Replace
"You can also [set your own WordPress installation](use-your-wordpress-installation.md). This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com. "
with
"You can also [configure your own WordPress installation](use-your-wordpress-installation.md) to be the data source. This can be hosted either locally on your development machine, or on a web-server. You can also use a site hosted on wordpress.com."